pkg/operator/csidriveroperator/csioperatorclient/cinder.go

```diff
@@ -15,26 +13,46 @@ const (
 	pairs := []string{
 		"${OPERATOR_IMAGE}", os.Getenv(envOpenStackCinderDriverOperatorImage),
 		"${DRIVER_IMAGE}", os.Getenv(envOpenStackCinderDriverImage),
 	}
 
-	return CSIOperatorConfig{
+	csiDriverConfig := CSIOperatorConfig{
 		CSIDriverName:   OpenStackCinderDriverName,
 		ConditionPrefix: "OpenStackCinder",
 		Platform:        configv1.OpenStackPlatformType,
-		StaticAssets: []string{
-			"csidriveroperators/openstack-cinder/02_sa.yaml",
-			"csidriveroperators/openstack-cinder/03_role.yaml",
-			"csidriveroperators/openstack-cinder/04_rolebinding.yaml",
-			"csidriveroperators/openstack-cinder/05_clusterrole.yaml",
-			"csidriveroperators/openstack-cinder/06_clusterrolebinding.yaml",
-		},
-		CRAsset:         "csidriveroperators/openstack-cinder/08_cr.yaml",
-		DeploymentAsset: "csidriveroperators/openstack-cinder/07_deployment.yaml",
 		ImageReplacer:   strings.NewReplacer(pairs...),
 		AllowDisabled:   false,
 	}
+
+	if !isHypershift {
+		csiDriverConfig.StaticAssets = []string{
+			"csidriveroperators/openstack-cinder/standalone/generated/v1_serviceaccount_openstack-cinder-csi-driver-operator.yaml",
+			"csidriveroperators/openstack-cinder/standalone/generated/rbac.authorization.k8s.io_v1_role_openstack-cinder-csi-driver-operator-role.yaml",
+			"csidriveroperators/openstack-cinder/standalone/generated/rbac.authorization.k8s.io_v1_rolebinding_openstack-cinder-csi-driver-operator-rolebinding.yaml",
+			"csidriveroperators/openstack-cinder/standalone/generated/rbac.authorization.k8s.io_v1_clusterrole_openstack-cinder-csi-driver-operator-clusterrole.yaml",
+			"csidriveroperators/openstack-cinder/standalone/generated/rbac.authorization.k8s.io_v1_clusterrolebinding_openstack-cinder-csi-driver-operator-clusterrolebinding.yaml",
+		}
+		csiDriverConfig.CRAsset = "csidriveroperators/openstack-cinder/standalone/generated/operator.openshift.io_v1_clustercsidriver_cinder.csi.openstack.org.yaml"
+		csiDriverConfig.DeploymentAsset = "csidriveroperators/openstack-cinder/standalone/generated/apps_v1_deployment_openstack-cinder-csi-driver-operator.yaml"
+	} else {
+		csiDriverConfig.StaticAssets = []string{
+			"csidriveroperators/openstack-cinder/hypershift/guest/generated/rbac.authorization.k8s.io_v1_clusterrole_openstack-cinder-csi-driver-operator-clusterrole.yaml",
+			"csidriveroperators/openstack-cinder/hypershift/guest/generated/rbac.authorization.k8s.io_v1_clusterrolebinding_openstack-cinder-csi-driver-operator-clusterrolebinding.yaml",
+			"csidriveroperators/openstack-cinder/hypershift/guest/generated/rbac.authorization.k8s.io_v1_role_openstack-cinder-csi-driver-operator-role.yaml",
+			"csidriveroperators/openstack-cinder/hypershift/guest/generated/rbac.authorization.k8s.io_v1_rolebinding_openstack-cinder-csi-driver-operator-rolebinding.yaml",
+			"csidriveroperators/openstack-cinder/hypershift/guest/generated/v1_serviceaccount_openstack-cinder-csi-driver-operator.yaml",
+		}
+		csiDriverConfig.MgmtStaticAssets = []string{
```

Comment:
The [k8s.io_v1_clusterrole_openstack-cinder-csi-driver-operator-clusterrole.yaml](https://github.com/openshift/cluster-storage-operator/pull/525/files#diff-4470c09bc16b29804d501f56214b9c3a7a4cd6c9967eff204b7163de28d69c2a) is not being loaded here in the management files, do we really need to generate it?

Reply:
Right you are. Fixed.